### PR TITLE
[rosa_hcp_migration] Updates XREF links

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR repairs the links to the **[Introduction to ROSA](https://82513--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_architecture/)** and **[Learning about ROSA](https://82513--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_learning/creating_cluster_workshop/cloud-experts-getting-started-hcp-for-hcp)** sections of the ROSA with HCP migration.

This PR also updates the **[ROSA with HCP quick start](https://82513--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_hcp/rosa-hcp-quickstart-guide.html)** guide.

### CODE REVIEW

File: modules/rosa-getting-started-deleting-a-cluster.adoc [hunk 1]
```diff
@@ -5,7 +5,12 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="rosa-getting-started-deleting-a-cluster_{context}"]
+ifdef::openshift-rosa-hcp[]
```
Comment (bhardesty) [inline modules/rosa-getting-started-deleting-a-cluster.adoc:8]: These two conditions seem to be superfluous. Whether the distro is openshift-rosa-hcp or not, the title is the same.

File: modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc [hunk 2]
```diff
@@ -23,21 +17,21 @@ endif::[]
...
 You can quickly create a
-ifdef::rosa-hcp[]
-{hcp-title}
-endif::rosa-hcp[]
-ifndef::rosa-hcp[]
-{product-title} (ROSA)
-endif::rosa-hcp[]
-cluster with the {sts-first} by using the default installation options. The following summary describes the default cluster specifications.
+ifdef::openshift-rosa-hcp[]
+{hcp-title} cluster by using the default installation options.
```
Comment (bhardesty) [inline modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc:21]: Should we be using `{hcp-title}` here? I would think it should just be `{product-title}`.

File: modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc [hunk 3]
```diff
@@ -23,21 +17,21 @@ endif::[]
...
+endif::openshift-rosa-hcp[]
+ifdef::openshift-rosa[]
+{product-title} (ROSA) cluster with the {sts-first} by using the default installation options.
+endif::openshift-rosa[]
+The following summary describes the default cluster specifications.
 endif::tf-classic,tf-hcp[]
 
-ifdef::rosa-hcp[]
+ifdef::openshift-rosa-hcp[]
 .Default {hcp-title} cluster specifications
```
Comment (bhardesty) [inline modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc:30]: Same comment as before.

File: modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc [hunk 4]
```diff
@@ -83,22 +77,22 @@ endif::rosa-hcp,tf-hcp[]
...
 |* Cloud storage is encrypted at rest
 * Additional etcd encryption is not enabled
-ifndef::rosa-hcp,tf-hcp[]
+ifdef::openshift-rosa,tf-classic[]
 * The default AWS Key Management Service (KMS) key is used as the encryption key for persistent data
-endif::rosa-hcp,tf-hcp[]
-ifdef::rosa-hcp,tf-hcp[]
+endif::openshift-rosa,tf-classic[]
+ifdef::openshift-rosa-hcp,tf-hcp[]
 * AWS Key Management Service (KMS) key encryption is not enabled by default.
```
Comment (bhardesty) [inline modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc:84]: Really nitpicky, but this prereq ends with a period and the others don't.

File: modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc [hunk 5]
```diff
@@ -139,12 +133,12 @@ ifndef::tf-classic,tf-hcp[]
 endif::tf-classic,tf-hcp[]
 * Host prefix: /23
 +
-ifdef::rosa-hcp[]
+ifdef::openshift-rosa-hcp[]
 [NOTE]
 ====
 When using {hcp-title}, the static IP address `172.20.0.1` is reserved for the internal Kubernetes API address. The machine, pod, and service CIDRs ranges must not conflict with this IP address.
```
Comment (bhardesty) [inline modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc:139]: Can we just remove "When using {hcp-title}"? We already know the customer will only be using ROSA HCP in this scenario.

File: nodes/index.adoc [hunk 6]
```diff
@@ -34,13 +34,13 @@ image::295_OpenShift_Nodes_Overview_1222.png[Overview of control plane and worke
...
-ifndef::openshift-rosa-hcp,openshift-rosa[]
+ifndef::openshift-enterprise,openshift-rosa-hcp,openshift-rosa[]
 * xref:../nodes/nodes/nodes-nodes-viewing.adoc#nodes-nodes-viewing-listing_nodes-nodes-viewing[List all the nodes in a cluster].
 * Get information about a node, such as memory and CPU usage, health, status, and age.
 * xref:../nodes/nodes/nodes-nodes-viewing.adoc#nodes-nodes-viewing-listing-pods_nodes-nodes-viewing[List pods running on a node].
-endif::openshift-rosa-hcp,openshift-rosa[]
+endif::openshift-enterprise,openshift-rosa-hcp,openshift-rosa[]
 
-ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+ifndef::openshift-enterprise,openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
```
Comment (bhardesty) [inline nodes/index.adoc:43]: I'm not sure I understand the conditionalization in this file. Some of it (like this instance) seems to exclude content from all OpenShift distros.

File: rosa_hcp/rosa-hcp-quickstart-guide.adoc [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+include::modules/rosa-getting-started-environment-setup.adoc[leveloffset=+1]
+[discrete]
+include::modules/rosa-getting-started-enable-rosa.adoc[leveloffset=+2]
+[discrete]
+include::modules/rosa-getting-started-install-configure-cli-tools.adoc[leveloffset=+2]
+
+.Next steps
+
+Before you can use the {cluster-manager} {hybrid-console-second} to deploy ROSA clusters, you must associate your AWS account with your Red{nbsp}Hat organization and create the required account-wide STS roles and policies. For information on how your AWS and Red Hat accounts interact, see the ROSA tutorial about SOMETHING and xref:../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-prerequisites.adoc#rosa-sts-understanding-aws-account-association_cloud-experts-deploying-application-prerequisites[Understanding AWS account association]
```
Comment (bhardesty) [inline rosa_hcp/rosa-hcp-quickstart-guide.adoc:22]: Do we really want "SOMETHING" here? :-)

**Review by bhardesty — COMMENTED** [review]
Overall looks great! I just have a few comments mostly related to conditionalization and attribute usage.

File: nodes/index.adoc (hunk 6 above)
Comment (EricPonvelle) [reply nodes/index.adoc:43]: These sections weren't working at all and I figured I'd push fixing it to another PR.

Comment (bhardesty) [reply nodes/index.adoc:43]: :+1: I'm currently working on the Nodes content. Perhaps that will automagically fix it. (We can always dream...)

**Pull request merged**